Select.js

```diff
@@ -239,7 +238,9 @@ define([
 						option.__dataItem = renderItem.__item; // __item is set by StoreMap.itemToRenderItem()
 						// to allow retrieving the option element from widget's selectedItems
 						// (which are data items, not render items).
-						option.__dataItem.__visualItem = option;
+						if (!option.__dataItem.__visualItem) {
```

Comment:
Why is it needed to change the priority between option.__dataItem.__visualItem and option ? Shouldn't option.__dataItem = renderItem.__item be changed too ?